functions/packages.sh

```diff
@@ -40,6 +40,50 @@ firemotd_setup() {
   fi
 }
 
+create_mta_config() {
+  TMP=/tmp/.$$
```

Comment:
If I understand the next lines correctly, this is a [security concern](https://en.wikipedia.org/wiki/Symlink_race) when using a temporary files to store passwords or other sensitive information.

Use [`mktemp`](https://www.gnu.org/software/autogen/mktemp.html) instead:

e.g.


```suggestion
  TMP="$(mktemp /tmp/.XXXXXXXXXX)"
```